dts/bindings/usb-c/usb-c-vbus-adc.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    This device is used to measure VBUS on a Type-C Port and to
+    discharge VBUS when needed.
+
+compatible: "usb-c-vbus-adc"
```

Comment:
should be `zephyr,usb-c-vbus-adc` (rename file as well).

Reply:
Done